flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
:-( I thought Java was "run everywhere"

Reply:
It uses (and bundles) [SWT](https://en.wikipedia.org/wiki/Standard_Widget_Toolkit) which links to GTK.

Reply:
It doesn't look like a build from source would be particularly difficult, assuming the existence of a maven manifest generator or some manual work to fake it… But I'm afraid I'm not about to work on it.

https://gitbox.apache.org/repos/asf?p=directory-studio.git;a=blob;f=build.sh;h=698c8e80d4a89c1aeddef7794831c7b3a39b359f;hb=HEAD